membership-common/src/main/scala/com/gu/zuora/soap/Client.scala

```diff
@@ -60,7 +60,8 @@ class Client(
       logger.info(
         s"Zuora SOAP call in environment ${apiConfig.envName}. Request info:\n${action.prettyLogInfo}. Is authentication defined: ${authentication.isDefined}",
       )
-    client.execute(request)
+    client
+      .execute(request)
```

Comment:
I don't think scalafmt is actually enforced on the membership-common code, but I did run it locally hence the random changes.